benchmarks/nemo/tracer_advection/scripts/acc_loops_trans.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
Since this isn't an executable script, I don't think you need this line?

Reply:
Removed